src/dist/replication/test/replica_test/unit_test/replica_test.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2017-present, Xiaomi, Inc.  All rights reserved.
+// This source code is licensed under the Apache License Version 2.0, which
+// can be found in the LICENSE file in the root directory of this source tree.
+
+#include <gtest/gtest.h>
+
+#include <dsn/utility/fail_point.h>
+#include "replica_test_base.h"
+#include <dsn/utility/defer.h>
+
+namespace dsn {
+namespace replication {
+
+class replica_test : public replica_test_base
```

Comment:
`replica_test` seems too common, could you please rename this class name?

Reply:
The class aims to test some common unit in future